README.md

```diff
@@ -24,3 +22,54 @@ Imagine that we can fetch from an API the details of all movies that were ever s
 **License**: `[<start year of the license>: Number, <end year of the license>: Number]`
 
 Write a JS Module that returns in what year we had the maximum number of movies on that platfrom
+
+# Solution Complexity
+
+## Time Complexity
+
+`O(nk)` - Where `n` is the number of elements in the array and `k` is the difference between the start year and the end year
```

Comment:
Is 'k' the same for every item in the array?

Make it k-max, to cover worst-case

Reply:
K isn't the same for every item, I didn't know how to represent k if k is variable for every item. 
Does writing k-max instead of k is the solution?

Reply:
Basically here I meant `k` as the difference b/w the smallest and the largest year present.